portfolio/src/main/webapp/index.html

```diff
@@ -44,9 +44,24 @@ <h3>Fun Facts</h3>
         <h3>Edinburgh</h3>
         <p>Here is a sketch of mine, showing some of my favourite places in Edinburgh.</p>
       </section>
+
       <div class="img-nonprofile">
         <img src= "/images/edinburgh-sketch.png" alt="Edinburgh Buildings Sketch" 
-        width=480px height=333px margin-right=15px>
+        width=480px height=333px margin-right=15px usemap="#buildingmap">
+        <map name="buildingmap">
+            <area shape="rec" coords="23,43,133,119" 
+            href="http://www.pilgrimbar.co.uk/" alt="Pilgrim Bar" target="_new">
```

Comment:
Using `target="_blank"` seems to be the preferable choice ([ref](https://stackoverflow.com/q/4964130))

Reply:
Thanks Kazi!